scipy/stats/_continuous_distns.py

```diff
@@ -1738,6 +1738,10 @@ def _munp(self, n, c):
     def _stats(self, c):
         return 0, None, 0, None
 
+    def _entropy(self, c):
+        h = -np.log(c / 2) + (c - 1) * np.euler_gamma / c + 1
```

Comment:
Or for simplicity and easy of maintenance,
```suggestion
        h = stats.weibull_min(c)._entropy() - np.log(0.5)
```

Tests can be simplified accordingly by comparing against the entropy of `weibull_min`.

If $f'(x)$ is the dweibull pdf and $f(x)$ is the weibull pdf, $f'(x) = 1/2 f(|x|)$. Then 
$h' = -\int_{-\infty}^\infty f'(x) \cdot \log(f'(x)) $ 
$h' =2 \cdot -\int_{0}^\infty 1/2 f(x) \cdot \log(1/2 f(x)) $
$h'= - \int_{0}^\infty f(x) \cdot \left(\log(1/2) + \log(f(x) \right)$  
$h' = -  \log(1/2) + h $

where $h$ is the weibull entropy.

Reply:
Thank you for trying it! Yes, sometimes I make mistakes. I have corrected the comment above. The idea was to use the private `_entropy` method, but yes, the argument `c` needs to be passed into the method rather than being used to freeze this distribution. Use of the private `_entropy` method avoids doubling the overhead incurred by the public `entropy`.